src/app/Http/Controllers/API/PollController.php

```diff
@@ -24,7 +25,7 @@ class PollController extends Controller
      */
     public function index(Presentation $presentation)
     {
-        //
+        return PollResource::collection($presentation->polls());
```

Comment:
you need to check if the logged user belongs to the presentation to avoid CSRF security issues

Reply:
Fixed in 742e1b4